packages/animation-curve-block/src/components/BlankSlate.tsx

```diff
@@ -0,0 +1,102 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useState } from 'react';
+
+import { IconPlus24, generateRandomId } from '@frontify/fondue';
+
+import { AnimationCurve, AnimationCurveType, BlankSlateProps, defaultAnimationCurveTypeValues } from '../types';
+import { AnimationCurveFlyout, CardText } from './';
+import { BlockInjectButton } from '@frontify/guideline-blocks-shared';
+
+export const BlankSlate = ({
+    appBridge,
+    content,
+    designTokens,
+    hasBorder,
+    lineColor,
+    endpointColor,
+    gridColor,
+    canvasHeight,
+    setLocalItems,
+    setBlockSettings,
+}: BlankSlateProps) => {
+    const height = canvasHeight !== 0 ? `${canvasHeight}px` : '82px';
+    const [title, setTitle] = useState('');
+    const [description, setDescription] = useState('');
+    const [isFlyoutOpen, setIsFlyoutOpen] = useState(false);
+
+    const defaultAnimationCurve: AnimationCurve = {
+        id: generateRandomId(),
+        title: '',
+        description: '',
+        animationFunction: {
+            type: AnimationCurveType.EaseInOut,
+            parameters: defaultAnimationCurveTypeValues[AnimationCurveType.EaseInOut],
+            duration: 1.0,
```

Comment:
you could also put this as a const

Reply:
added a constant `DEFAULT_ANIMATION_FUNCTION`